components/new/create-form.js

```diff
@@ -133,6 +133,8 @@ function CreateForm({defaultCommune}) {
             id='commune'
             initialSelectedItem={defaultCommune}
             label='Commune'
+            hint='Si vous ne trouvez pas votre commune, essayez d’ajouter le code département'
```

Comment:
```suggestion
            hint='Si vous ne trouvez pas votre commune, nous vous recommandons de renseigner le code département'
```
Un peu long peut-être ?

Je ne trouve pas le terme "essayez" adapté. Ça pourrait donner l'impression que nous ne sommes pas sûr.